scripts/serpicoInstall.sh

```diff
@@ -0,0 +1,55 @@
+# serpicoInstall.sh
+
+#!/bin/bash
+
+username="r3b00t"
```

Comment:
username should not be hard coded.  It should be something like username=$(whoami)

Reply:
Good point. Will change and re-submit.